src/item.c

```diff
@@ -93,7 +95,9 @@ void CopyItemNameHandlePlural(u16 itemId, u8 *dst, u32 quantity)
     if (quantity < 2)
         return;
 
-    if (ItemId_GetPocket(itemId) == POCKET_BERRIES)
+    if (DoesItemHavePluralName(itemId))
+        StringCopy(dst, ItemId_GetPluralName(itemId));
+    else if (ItemId_GetPocket(itemId) == POCKET_BERRIES)
         GetBerryCountString(dst, gBerries[itemId - FIRST_BERRY_INDEX].name, quantity);
```

Comment:
Since berries have the plural in `gItems` now, this function could/should probably be deprecated

Reply:
I agree we should delete these two lines.
I agree that we should deprecate `GetBerryCountString`
I think that the deprecation is out of scope for this specific PR and should be captured in an issue.

Reply:
we talked offline, will do later
https://github.com/rh-hideout/pokeemerald-expansion/issues/4010